content/components/banner.mdx

```diff
@@ -10,34 +10,52 @@ import ComponentLayout from '~/src/layouts/component-layout'
 export default ComponentLayout
 import {AccessibilityLink} from '~/src/components/accessibility-link'
 
+Banner can be used in one of two ways:
+
+1. To highlight prominent information on a page.
+2. To communicate feedback in response to a user action.
+
 ## Accessibility
 
-### Focus management vs. Live region announcement
+### Feedback Banners
 
-Banner can be used in one of two ways:
+Banners that are used to communicate feedback require extra accessibility considerations to ensure they are surfaced immediately to all users including those using assistive technologies. This relates to [WCAG 4.1.3 Status Messages](https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html).
+
+Example scenarios that require extra considerations include:
+
+* When a user submits a form, an error banner is shown with a list of form errors that need to be corrected.
+* When a user submits a form, the submission successfully goes through and a success banner is shown.
+* When a user saves a setting, an error banner is shown to communicate that an unexpected error has occured and the setting was not saved.
+* When a user toggles an option, a warning banner is shown to warn the user of potential consequences.
+
+Banners used in these scenarios can be made more accessible through either a **focus management** or a **live region announcement** technique:
+
+* **Focus management** would involve moving focus to the newly shown Banner. This is the most reliable way to ensure that a user is aware of the new content. This is the preferred method for critical Banner scenarios (with exceptions).
+* **Live region announcements**: A live region announcement can be used to announce the new content to screen reader users. This method is less reliable than focus management, but may be the preferred method for non-critical information (with exceptions).
+
+Here are some questions to consider when deciding which method to use:
+
+#### How critical is the information?
+
+Critical information (such as an error) should be made as discoverable as possible. Though moving focus to the Banner can be considered more disruptive than a live region announcement, it's more important that a user is able to discover the Banner so they can take an appropriate action. In most critical scenarios, using focus management is the best approach.
+
+On the other hand, for communicating newly shown non-critical information (such as a success), a live region announcement is sufficient in **most** scenarios. A live region announcement can sometimes fail to fire and can easily be missed, so it should be reserved for use on non-critical information where there are no consequences if a user misses it.
+
+#### Where is a user's focus when the banner appears?
```

Comment:
I realized I accidentally deleted this question in my previous PR, so I added it back.